features/step_definitions/git-flow.js

```diff
@@ -121,7 +121,7 @@ defineSupportCode(({ Before, Given, When, Then }) => {
     When(/^new changes are pushed to that pull request$/, {
         timeout: TIMEOUT
     }, function step() {
-        return this.promiseToWait(3) // Find & save the previous build
+        return this.promiseToWait(10) // Find & save the previous build
```

Comment:
This is really just kicking the can further down the road. It isn't solving the underlying problem. A better solution would be to have an exponential-backoff retry to get the right data.

Reply:
Agree, should record the `buildId` for the previous build and make sure the new one has a different `buildId`. I will create a issue for this. This PR is to unblock the pipeline for now.